arkane/encorr/reference.py

```diff
@@ -165,6 +166,40 @@ def as_dict(self):
 
         return dictionary
 
+    def is_isomorphic(self, species: Union[ArkaneSpecies, Species], generate_resonance_structures: bool = True) -> bool:
```

Comment:
Regarding my previous comment about the issues with `is_isomorphic`, maybe the charge comparison can be built into this function?